x/treasury/internal/types/events.go

```diff
@@ -1,5 +1,13 @@
 package types
 
 const (
+	EventTypeMsgSend = "msg_send"
```

Comment:
```suggestion
	EventTypeSend = "send"
```

Reply:
Would prefer if we stick to excluding the 'Msg' part

Reply:
Ok. updated the code to EventTypeSend  = "send"